spk/sonarr/Makefile

```diff
@@ -38,6 +38,12 @@ POST_STRIP_TARGET = sonarr_extra_install
 # TMPDIR is used for sonarr built in backup and update
 USE_ALTERNATE_TMPDIR = 1
 
+include ../../mk/spksrc.common.mk
+ifeq ($(call version_lt, ${TCVERSION}, 7.0),1)
+# use the updated libstdc++ library on DSM < 7.0.
```

Comment:
as you can see in https://github.com/dotnet/runtime/issues/33384
dotnet core requires at least version 6.0.22 (but DSM 6 has version 6.0.20 only)
```suggestion
# dotnet requires libstdc++ version >= 6.0.22
```

I prefer to document the required version here (for the rare case that synology updates the version of libstdc++ in DSM 6).